Contribute/get-started-setup-tools.md

```diff
@@ -19,40 +21,54 @@ This article describes the steps to interactively install Git client tools and V
 
-## Install Git client tools
+## Install Git
 
-This install includes the Git version control system and Git Bash, the command-line app that you use to interact with your local Git repository.
+When installing Git, a version control system, on Windows, it includes Git Bash, a command-line application that you can use to run commands and interact with your local Git repository.
 
-### Install Git client tools for Windows
+> [!TIP]
+> If you already have Visual Studio Code installed you can set it as Git's default editor during setup.
+
+### Install Git on Windows
 
 [!INCLUDE[Install Git client tools for Windows](~/guide/help-content/includes/proc-contribute-install-git-client-tools.md)]
 
+### Install Git on Linux and macOS
+
+* Git on macOS is provided as part of the Command Line Tools for Xcode. Simply run `git` from Terminal, and you will be prompted to install the command line tools if needed. You can learn more about installing [Git on macOS](https://git-scm.com/download/mac) from the Software Freedom Conservancy.
+* [Git on Linux and Unix](https://git-scm.com/download/linux)
 
-### Install Git client tools for Mac and Linux
+### Configure Git
 
-* Git for Mac is provided as part of the Xcode Command Line Tools. Simply run `git` from the command line. You will be prompted to install the command line tools if needed. You can also download [Git for Mac](https://git-scm.com/download/mac) from the Software Freedom Conservancy.
-* [Git for Linux and Unix](https://git-scm.com/download/linux)
+After installation, Git comes ready-to-go, with the exception of two important configurations. For your changes to be properly tracked and attributed, you must set a local username and email. **Do not skip this step.**
 
-Follow the instructions for your chosen client for installation and configuration.
+The following commands set the default username and email for every repository on your device, not just Microsoft repositories. For a more detailed guide on configuring Git, see GitHub's [Setting up Git](https://docs.github.com/articles/set-up-git#setting-up-git) documentation.
 
-In the next article, you will [Set up a local Git repository](get-started-setup-local.md).
+```bash
+git config --global user.name "First Last"
+```
 
-   Additional Git resources are available here: [Git terminology](https://help.github.com/articles/github-glossary) | [Git basics](https://git-scm.com/book/en/v2/Getting-Started-Git-Basics) | [Learning Git and GitHub](https://help.github.com/articles/good-resources-for-learning-git-and-github/)
+```bash
+git config --global user.email "email@example.com"
```

Comment:
This new addition can cause some privacy/SPAM issues - putting your email address out there on the internet.  This is another way to get a no-reply email.

```
In a browser, go to GitHub.com and follow these steps to configure your Git name and email address.

   1. Sign in to [GitHub](https://github.com/) using your GitHub account.
   1. From your **Profile** drop-down select **Settings**.
   1. Select **Emails** from the left nav. Your email settings appear. Shortcut: [https://github.com/settings/emails](https://github.com/settings/emails).
   1. Make sure this option is checked: **Keep my email addresses private**
   1. Make a note of your *noreply* email address (`<UID>+<github_name>@users.noreply.github.com`).
   1. On your computer, open Git bash to configure your Git details using your noreply email address.

      ```bash
      git config --global user.name "My Name"
      git config --global user.email "<noreply_email_address>"
      ```

   1. List your local settings to ensure the user.name and user.email values are correct.

      ```bash
      git config --list
      ```
```

Reply:
I didn't think of that. That's a good call. I want to add this, but I'm unsure how to move forward. Here are a few ideas I had:
1. Replace the current config with the GitHub noreply config?
2. Provide both the current config and the GitHub noreply config. Adding a note explaining why one might choose noreply over traditional.
3. Remove this altogether and just link to [GitHub's docs for setting a username and acquiring a noreply email. ](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address)

For example, I could do something like:
```markdown
### Configure Git
...

#### NoReply Email

You might want to keep your email address hidden for privacy/SPAM issues. If not, skip this section.

1.  Sign in to [GitHub](https://github.com/) using your GitHub account.
2. ... [acquire no-reply steps]

#### Configure Email

git config --global user.email "email@example.com"

```

Let me know what you think.